src/mlpack/tests/sgd_test.cpp

```diff
@@ -39,6 +41,23 @@ BOOST_AUTO_TEST_CASE(SimpleSGDTestFunction)
   BOOST_REQUIRE_SMALL(coordinates[2], 1e-7);
 }
 
```

Comment:
I also thought about this, but how could we extract the gradient history from SGD optimizer object?